src/styles/blocks/cards.scss

```
@@ -0,0 +1,137 @@
+.cards {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
+  gap: 44px 46px;
+  margin: 50px 40px;
+
+  @media (max-width: 1023.98px) {
```

Use only integer numbers
```suggestion
  @media (max-width: 1024px) {
```